ui/lib/pki/addon/components/pki-tidy-form.hbs

```diff
@@ -27,6 +27,18 @@
       />
     {{/let}}
   {{/if}}
+  {{#each @tidy.autoTidyEnabledFields as |fieldGroup|}}
+    {{#each-in fieldGroup as |group fields|}}
+      {{#if (and (eq @tidyType "auto") @tidy.enabled)}}
```

Comment:
Since we already check for `@tidyType "auto"` above, I might refactor this a bit so we can reuse that logic, the whole thing would look something like this. 

(I intentionally removed the check for `and....this.intervalDurationAttr` as I'm not sure why it was there in the first place... 😅 that value should always exist since it's computed in the model)
```hbs
  {{#if (eq @tidyType "auto")}}
    {{#let this.intervalDurationAttr as |attr|}}
      <TtlPicker
        data-test-input={{attr.name}}
        @onChange={{fn this.handleTtl attr}}
        @label={{attr.options.label}}
        @labelDisabled={{attr.options.labelDisabled}}
        @helperTextDisabled={{attr.options.helperTextDisabled}}
        @helperTextEnabled={{attr.options.helperTextEnabled}}
        @initialEnabled={{get @tidy attr.options.mapToBoolean}}
        @initialValue={{get @tidy attr.name}}
      />
    {{/let}}

    {{#if @tidy.enabled}}
      {{#each @tidy.autoTidyConfigFields as |field|}}
         <FormField @attr={{get @tidy.allByKey field}} @model={{@tidy}} />
      {{/each}}
    {{/if}}
  {{/if}}

Reply:
Fixed alignment in https://github.com/hashicorp/vault/pull/28488/commits/24332c27ef1eaa2ffa30d78208adf308069d870a